exercises/tuples-lists-dictionaries/challenge_069.py

```diff
@@ -0,0 +1,45 @@
+# TUPLES, LISTS AND DICTIONARIES
+
+# Challenge 069
+"""
+Create a tuple containing the names of five countries and display the whole tuple.
+Ask the user to enter one of the countries that have been shown to them and then display
+the index number (i.e. position in the list) of that item in the tuple.
+"""
+
+country_names = ("England", "Italy", "Spain", "France", "Portugal")
+
+print(country_names)
+
+
+# Ask for user input and ensuring the user entering an integer
+def get_user_input(message):
+    while True:
+        user_input = input(message)
+        if user_input.isnumeric():
```

Comment:
Here I think you just need to check whether the input is in the tuple and loop until the user enters something that is present

Reply:
simplified this logic